sim/Makefile

```diff
@@ -42,12 +42,22 @@ SBT ?= java $(JAVA_ARGS) -jar $(rocketchip_dir)/sbt-launch.jar
 # Manage the FIRRTL dependency manually
 FIRRTL_SUBMODULE_DIR ?= $(chipyard_dir)/tools/firrtl
 FIRRTL_JAR ?= $(chipyard_dir)/lib/firrtl.jar
-$(FIRRTL_JAR): $(shell find $(FIRRTL_SUBMODULE_DIR)/src/main/scala -iname "*.scala")
+FIRRTL_TEST_JAR ?= $(chipyard_dir)/test_lib/firrtl.jar
+
+firrtl_srcs := $(shell find $(FIRRTL_SUBMODULE_DIR) -iname "*.scala")
+
+$(FIRRTL_JAR): $(firrtl_srcs)
```

Comment:
I think the way this recipe is conservative but fast if unneeded is good.

Reply:
Would it make sense to copy what firrtl does in build-scala?